start_meshnet.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
comaybe should be in contrib dir

Reply:
Removed this one completely since it's not finished anyway. I'll find time to add it in the future.